testing/sdk_utils.py

```diff
@@ -1,4 +1,5 @@
 import sys
+
```

Comment:
Nit (you REALLY don't need to respond to this): I would generally avoid making edits (even if stylistically correct) in unrelated files just to absolutely limit the sticker shock of any given diff. We will definitely need to do a huge style PR later